org.eclipse.xtext.xtext.generator/src/org/eclipse/xtext/xtext/generator/XtextGeneratorTemplates.xtend

```diff
@@ -466,7 +466,7 @@ class XtextGeneratorTemplates {
 			
 				@Override
 				protected �'org.osgi.framework.Bundle'.typeRef� getBundle() {
```

Comment:
same here

Reply:
ok, will change that, too.

Reply:
What means 'same here'? The method signature does not change.

Reply:
use class instead of string